eth/ulc_config.go

```diff
@@ -1,36 +1,9 @@
 package eth
 
-import (
-	"github.com/ethereum/go-ethereum/log"
-	"github.com/ethereum/go-ethereum/p2p"
-	"github.com/ethereum/go-ethereum/p2p/discover"
-)
-
-const defaultUTCMinTrustedFraction = 75
+const DefaultUTCMinTrustedFraction = 75
 
 // ULCConfig is a Ultra Light client options.
 type ULCConfig struct {
-	TrustedNodes       []string `toml:",omitempty"` // A list of trusted servers
+	ULCTrustedNodes    []string `toml:",omitempty"` // A list of trusted servers
```

Comment:
Why prefix `ULC` here? We are already inside the `ULCConfig`.

Reply:
I think it's because there is trusted nodes field in p2p.Server and p2p.Config. There is a lot of embedding of these structures.

Reply:
Maybe just `TrustedServers`?